packages/next-auth/src/authOptions.ts

```diff
@@ -55,7 +55,7 @@ export const authOptions: AuthOptions = {
       };
     },
     redirect() {
-      return env.KAMPUS_ENV === "production" ? "https://dev.kamp.us" : "http://localhost:3000";
+      return env.KAMPUS_ENV === "production" ? "https://dev.kamp.us" : "http://localhost:3001";
```

Comment:
I think this should not be changed as it redirects to the homepage

Reply:
oh yeah, this changes it back to redirecting to pasaport, let's keep it as `3000` here as the default it being the homepage of `apps/kampus` makes sense